pkg/sql/sem/builtins/geo_builtins_test.go

```diff
@@ -103,3 +104,237 @@ func TestGeoBuiltinsPointEmptyArgs(t *testing.T) {
 		})
 	}
 }
+
+type singleArgBuiltinFixture struct {
```

Comment:
can you move these to the `geospatial` logic test?

we can ideally test this very easily with
```
SELECT description st_xmin(p), st_xmax(...), ... FROM ( VALUES (
   ('point', 'POINT(...)'),
   ....
)
```